api/ceramic.yaml

```diff
@@ -632,3 +679,33 @@ components:
         name:
           type: string
           description: Name of the Ceramic network
+    StreamState:
+      title: State of a Ceramic stream
+      description: The state of a Ceramic stream as defined by the stream type aggregation and conflict resolution rules.
+      type: object
+      required:
+        - id
+        - event_cid
+        - controller
+        - dimensions
+        - metadata
```

Comment:
what's the difference between dimensions and metadata?

Reply:
dimensions are static for the stream, metadata is mutable over the lifetime of the stream. shouldIndex is the only metadata field currently